src/components/Monster/Monster.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
-import style from './Monster.module.css';
+import { connect } from 'react-redux';
+import styles from './Monster.module.css';
+import classNames from 'classnames/bind';
+
+const cx = classNames.bind(styles);
+
 
 export class Monster extends React.Component {
     render() {
```

Comment:
Since this component only has a `render`, it could be a function component.

Reply:
done